docs/add-new-token.md

```diff
@@ -0,0 +1,158 @@
+# Adding New Token to the List
+
+The goal of this guide is walk you through the steps needed to add a new token to the token list.
+
+## Before you begin
+
+You need to have some informations about the token you want to add to the list before you begin this tutorial and they are:
+
+1. Chain ID
+
+The new token can be added to testnet and/or mainnet and the chain ID for each is the following:
+
+```json
+Testnet: 743111
+Mainnet: 43111
+```
+
+> If you want to add the token to both testnet and mainnet you need to execute these steps one time for each chain (one for testnet and another for mainnet).
+
+1. Token Address
+
+The new token's smart contract must already be deployed in the chain you want to add it and this process will provide a Token Address like the one below:
+
+[0x8970a6A9Eae065aA81a94E86ebCAF4F3d4dd6DA1](https://explorer.hemi.xyz/address/0x8970a6A9Eae065aA81a94E86ebCAF4F3d4dd6DA1)
+
+> You can get this information from the tokens section of Hemi Explorer.
+
+## Step 1 - Run the Add Token Script
+
+There is a script that automates the process of adding new tokens to the list and you just need use the `chain-id` and `address` you collected in the previous step:
+
+```sh
+node scripts/add-token <chain-id> <address>
+```
+
+> You need to install the project dependencies before running this command with `npm install`.
+
+The script will automatically add the information about the new token to [./src/hemi.tokenlist.json](../src/hemi.tokenlist.json) file as shown below:
+
+```json
```

Comment:
Use "jsonc" if you need to add comments to the JSON block. Do the same for the next code blocks too.

```suggestion
```jsonc
```

Reply:
fixed